core/commands/check_relevance_command.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'base_command'
+
+# This class checks relevance network config
+class CheckRelevanceCommand < BaseCommand
+  def self.synopsis
+    'Check relevance network config'
+  end
+
+  def execute
+    system 'scripts/check_network_config.sh ' + @args
```

Comment:
You must recreate the logic of this script inside the execute method. It is wrong to use several scripting languages to achieve the result. Also we can not be sure about the version of the SH interpreter that is installed, but we control the Ruby interpreter.

Reply:
Please also note that the execute method must return some sort of Result. Either `Result.ok()`, or `Result.error()`.

Reply:
I have checked the script, it currently uses heuristics to find out whether file is relevant or not. This is not appropriate. The correct approach would be to read the file, then try to connect to all the nodes that are specified inside the _network_config file. If all connections are successful, then the file is OK, otherwise - not.